v_5_2_0/files/k8s-resource/calico-policy-only.yaml

```diff
@@ -495,6 +495,10 @@ spec:
             # Disable IPv6 on Kubernetes.
             - name: FELIX_IPV6SUPPORT
               value: "false"
+            - name: FELIX_IPTABLESMANGLEALLOWACTION
+              value: Return
+            - name: FELIX_INTERFACEPREFIX
+              value: eni
```

Comment:
I'm pretty sure that our interfaces are not named `eni`. the right prefix here is `cali`